certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ID of the contract to add to the bundle",
+                  "x-ms-summary": "Contract ID"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/BundleResponse"
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/bundle/{id}/transaction/start": {
+      "post": {
+        "summary": "Send a bundle for signature",
+        "description": "Send a bundle for signature",
+        "operationId": "SendBundleForSignature",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Bundle ID",
+            "description": "ID of the bundle to send for signature",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transaction information",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Says if transaction has been sent or not"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/bundle/{id}/transaction/cancel": {
+      "post": {
+        "summary": "Withdraw a bundle",
+        "description": "Withdraw all contract in a bundle",
+        "operationId": "WithdrawBundle",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Bundle ID",
+            "description": "ID of the bundle to withdraw all contract",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Says if bundle was withdrawed"
```

Comment:
@QwerioTEAM all files should be free of grammatical errors. please change "withdrawed" to "withdrawn"